projects/demo-maps/src/app/route-components/route-example-custom-layers/resources/features.ts

```diff
@@ -0,0 +1,366 @@
+export const windMeasurements = {
```

Comment:
If you create a own file for this maybe we can outsource all data for Vector Layers?

Reply:
Sure thing! Will do!

Reply:
Done!